src/main/java/com/bookclub/controller/BooksController.java

```diff
@@ -53,6 +60,20 @@ public BooksController() {
      */
     @FXML
     public void initialize() {
+
+        // set genre combo box
+        for (Genre genre : Genre.values()) {
+            genreComboBox.getItems().add(genre.getDisplayName());
+        }
+
+        //Set genre search combo box
+        // Add "All" as the first option
+        searchGenreComboBox.getItems().add("All");
+        // Populate the combo box with genres from the Genre enum
+        for (Genre genre : Genre.values()) {
+            searchGenreComboBox.getItems().add(genre.getDisplayName());
+        }
+
```

Comment:
This can all be combined was more my suggestion

```suggestion
        searchGenreComboBox.getItems().add("All"); // Add "All" as the first option

        for (Genre genre : Genre.values()) {
            genreComboBox.getItems().add(genre.getDisplayName());
            searchGenreComboBox.getItems().add(genre.getDisplayName());
        }
```

Reply:
Done